metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/indexbuilder/ESIndexBuilder.java

```diff
@@ -220,6 +224,27 @@ public void buildIndex(ReindexConfig indexState) throws IOException {
     }
   }
 
+  public String reindexInPlaceAsync(String indexAlias, @Nullable QueryBuilder filterQuery, BatchWriteOperationsOptions options)
```

Comment:
This is very much just for timeseries, I'd say we should name it differently to make that clear. It also then makes since that a timeseries method would take a timeseries package options parameter.